src/CaptureClient/include/CaptureClient/CaptureListener.h

```diff
@@ -59,6 +60,8 @@ class CaptureListener {
       orbit_grpc_protos::LostPerfRecordsEvent lost_perf_records_event) = 0;
   virtual void OnOutOfOrderEventsDiscardedEvent(
       orbit_grpc_protos::OutOfOrderEventsDiscardedEvent out_of_order_events_discarded_event) = 0;
+
+  [[nodiscard]] virtual ApiEventIdSetter& GetApiEventIdSetter() = 0;
```

Comment:
I don't think this belongs here. I clearly think that this should be a member of either ApiEventProcessor OR CaptureEventProcessor.

Reply:
To `ApiEventProcessor` it goes.